src/main/flight/pid.c

```diff
@@ -167,7 +168,7 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     static firFilterDenoise_t denoisingFilter[2];
     static pt1Filter_t pt1FilterYaw;
 
-    uint32_t pidFrequencyNyquist = (1.0f / dT) / 2; // No rounding needed
+    const uint32_t pidFrequencyNyquist = (1.0f / dT) / 2; // No rounding needed
```

Comment:
I'm here as usual to tell someone that `1  / dT` produces the same code since operands of `operator /` undergo a integral-to-floating conversion :-)